.github/workflows/update_frontend.yml

```diff
@@ -0,0 +1,81 @@
+name: Update frontend
+
+on:
+  schedule: # once a day
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  check-version-and-create-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Get latest frontend release
+        id: latest_frontend_version
+        uses: abatilo/release-info-action@v1.3.3
+        with:
+          owner: home-assistant
+          repo: frontend
+      - name: Check if version is up to date
+        id: check_version
+        run: |
+          LANDING_PAGE_VERSION=$(cat .ha-frontend-version)
+          LATEST_VERSION=${{ steps.latest_frontend_version.outputs.latest_tag }}
+          echo "LANDING_PAGE_VERSION=$LANDING_PAGE_VERSION" >> $GITHUB_ENV
+          echo "LATEST_VERSION=$LATEST_VERSION" >> $GITHUB_ENV
+          if [[ ! "$LANDING_PAGE_VERSION" < "$LATEST_VERSION" ]]; then
+            echo "Frontend version is up to date"
+            echo "skip=true" >> $GITHUB_OUTPUT
+          fi
+      - name: Check if there is no open PR with this version
+        if: steps.check_version.outputs.skip != 'true'
+        id: check_existing_pr
+        run: |
+          PR=$(gh pr list --state open --base main --json title --search "Autoupdate frontend to version $LATEST_VERSION")
+          if [[ "$PR" != "[]" ]]; then
+            echo "Skipping - There is already a PR open for version $LATEST_VERSION"
+            echo "skip=true" >> $GITHUB_OUTPUT
+          fi
+      - name: Get old version PRs
+        if: steps.check_version.outputs.skip != 'true' && steps.check_existing_pr.outputs.skip != 'true'
+        id: get_old_prs
+        run: |
+          OLD_PRS=$(gh pr list --state open --base main --json number --search "Autoupdate frontend to version")
+          echo "OLD_PRS=$OLD_PRS" >> $GITHUB_ENV
+          if [[ "$OLD_PRS" != "[]" ]]; then
+            echo "old_prs_found=true" >> $GITHUB_OUTPUT
+          fi
+      - name: Close existing PRs of older version
+        if: steps.check_version.outputs.skip != 'true' && steps.check_existing_pr.outputs.skip != 'true' && steps.get_old_prs.outputs.old_prs_found == 'true'
+        run: |
+          apt-get install -y jq
+          for pr_number in $(jq -r '.[] | .number' <<< $OLD_PRS); do
+            gh pr close $pr_number --delete-branch --comment "Closed automatically because a newer frontend version is available"
+          done
```

Comment:
Is this required?

I use the same GitHub action here: https://github.com/home-assistant-libs/chip-wheels/blob/main/.github/workflows/dependency-update.yaml#L43-L58

And the outcome is a continuously updated PR, which to me looks fine: https://github.com/home-assistant-libs/chip-wheels/pull/117

Reply:
Nice, so we just have to set a static name for the branch?